Validation/RecoTrack/plugins/TrackingNtuple.cc

```diff
@@ -674,6 +674,7 @@ class TrackingNtuple : public edm::one::EDAnalyzer<edm::one::SharedResources> {
   const bool includeTrackingParticles_;
   const bool includeOOT_;
   const bool keepEleSimHits_;
+  const bool saveModuleType_;
```

Comment:
I think that this is a small enough addition that it can be just added as is

Reply:
Sure, I removed it as an option and this is added by default now.